pyirf/io/io.py

```diff
@@ -86,13 +86,3 @@ def get_simu_info(filepath, particle_name, config={}):
     print(cfg)
 
     return config
-
-
-
-# def get_resource(resource_name):
```

Comment:
This was added by @vuillaut , but I didn't see it used anywhere else is it stale code?

Reply:
It was completely commented out, so by definition not used.